.storybook/static/HCB_css_variables.css

```diff
@@ -1,494 +0,0 @@
-/**
```

Comment:
I've deleted all these storybook/static files a couple times now 🤣  They keep sneaking back in. We reference all of these from `sap-theming` now.